dts/arm/silabs/efr32mg21.dtsi

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020 TriaGnoSys GmbH
+ * Copyright (c) 2021 Safran Passenger Innovations Germany GmbH
```

Comment:
Don't remove the existing copyright; add a new line instead

Reply:
We went through a name change between 2020-2021 that's why I though I could update it but, it seems we better keep track of them, I didn't know about that. Thanks.